molgenis-data-excel/src/main/java/org/molgenis/data/excel/simple/ExcelWriter.java

```diff
@@ -0,0 +1,86 @@
+package org.molgenis.data.excel.simple;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Stream;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.molgenis.data.excel.simple.exception.MaximumSheetNameLengthExceededException;
+
+public class ExcelWriter implements AutoCloseable {
+
+  public static final int MAXIMUM_SHEET_LENGTH = 31;
```

Comment:
There is no restriction on worksheet name length in the standard ECMA-376 4th edition Part 1, so I propose to move this check to the class using the writer.

On a side note do you know if this limit is also an issue in the latest Excel version?

Reply:
It's an issue in the apache-poi library we use